database/entity/Transaction.ts

```diff
@@ -1 +1 @@
-export { Transaction } from './0066-x-community-sendcoins-transactions_table/Transaction'
```

Comment:
I think you base of work is outdated, because the master branch contains the following version:
0068-community_tables_public_key_length